spec/system/jobseekers/jobseekers_can_add_job_preferences_to_their_profile_spec.rb

```diff
@@ -0,0 +1,143 @@
+require "rails_helper"
+
+RSpec.describe "Jobseekers can add job preferences to their profile" do
+  let!(:jobseeker) { create(:jobseeker, :with_profile) }
+
+  before do
+    login_as(jobseeker, scope: :jobseeker)
+  end
+
+  after { logout }
+
+  describe "changing job preferences" do
+    context "when adding job preferences" do
+      before do
+        visit jobseekers_profile_path
+        create(:job_preferences, completed_steps: [], builder_completed: false, jobseeker_profile: jobseeker.jobseeker_profile)
+      end
+
+      it "allows jobseekers to add job preferences" do
+        click_on "Add job preferences"
+
+        # submit empty form
```

Comment:
Question: Weren't we trying to move away from doing all these error checks as system tests? They add-up quite a bit of time to the test runs and could be checked at lower-level testing.

Reply:
Ah, I didn't realise we had agreed on that. Have removed them! (We already have form tests for these, I just added this for completeness)